src/config.ts

```diff
@@ -14,35 +14,36 @@ interface Config {
     };
 }
 
-const configPath = process.env.CONFIG_PATH ? process.env.CONFIG_PATH : '/etc/reviewer/config.json';
-const config: Config = JSON.parse(readFileSync(configPath, 'utf8'));
-
-const knexConfig: KnexConfig = {
-    client: 'pg',
-    // In production we should use postgres pools.
-    connection: {
-        host: config.database.host,
-        database: config.database.database,
-        user: config.database.username,
-        password: config.database.password,
-        port: config.database.port,
-    },
-};
-
-const eventConfig: EventConfig = {
-    url: config.rabbitmq_url,
-};
-
 export type ConfigType = {
     port: number;
     knex: KnexConfig;
     event: EventConfig;
 };
 
-export const serviceConfig: ConfigType = {
-    port: Number(config.port) || 3004,
-    knex: knexConfig,
-    event: eventConfig,
+export const serviceConfig = (): ConfigType => {
```

Comment:
Should we rename this to be `getServiceConfig` to make it more implicit it needs to be invoked?